core/src/components/label/test/a11y/label.e2e.ts

```diff
@@ -82,3 +83,28 @@ configs({ directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
     });
   });
 });
+
+configs({ directions: ['ltr'], modes: ['md'], themes: ['light', 'dark'] }).forEach(({ title, config }) => {
+  test.describe(title('label: a11y for ion-color()'), () => {
+    test('should not have accessibility violations when focused', async ({ page, pageUtils }) => {
+      /**
+       * All page content should be contained by landmarks (main, nav, etc.)
+       * By containing the badge in a main element, we can avoid this violation.
+       */
+      await page.setContent(
+        `
+          <main>
+            <ion-input label="Stacked" label-placement="stacked"></ion-input>
```

Comment:
This does not use `ion-label` internally. You'll need to use `ion-label` directly instead.

```html
<ion-label color="primary">Label</ion-label>
```

Reply:
The modern syntax doesn't use `<ion-label>` so I opted to test when the label is [stacked/floating](https://github.com/ionic-team/ionic-framework/blob/c5a67cdabe7c0b03042161f2d467db877614e748/core/src/components/input/input.md.scss#L71-L74). Since the goal was to test the label, I had the test in this file instead of input.

Reply:
The goal of this task is to test `ion-label`, not labels in general. We already verify that the label in `ion-input` has sufficient contrast in https://github.com/ionic-team/ionic-framework/blob/7c1f8cc8afb43dea2f4cc109c88d06e26fc60276/core/src/components/input/test/a11y/input.e2e.ts#L37-L43

Reply:
Understood, change made in https://github.com/ionic-team/ionic-framework/pull/28640/commits/ae56a3c0506f976baaae9266f4d36c96bcf4a8b6